velox/functions/lib/TimeUtils.h

```diff
@@ -104,4 +106,77 @@ struct InitSessionTimezone {
     timeZone_ = getTimeZoneFromConfig(config);
   }
 };
+
+// Return the first `fisrtDayOfWeek` in the year.
+// getFirstDayOfWeek() use 1-based weekday number:
+//   Sunday - 1, Monday - 2, ..., Saturday - 7
+FOLLY_ALWAYS_INLINE
+uint32_t getFirstDayOfWeek(int32_t y, uint32_t firstDayOfWeek) {
```

Comment:
Would you add comments for the input parameters?

> Return the first `fisrtDayOfWeek` in the year

Perhaps also revise this comment to make it more clear.